docs/reference/cluster/nodes-stats.asciidoc

```diff
@@ -747,6 +746,154 @@ must be readable from `/sys/fs/cgroup/cpu` and `/sys/fs/cgroup/cpuacct`.
+(integer)
+Amount of non-heap memory available, in bytes.
+
+`jvm.mem.pools.young.used_in_bytes`::
+(integer)
+Memory, in bytes, used by the young generation heap.
+
+`jvm.mem.pools.young.max_in_bytes`::
+(integer)
+Maximum amount of memory, in bytes, available for use by the young generation
+heap.
+
+`jvm.mem.pools.young.peak_used_in_bytes`::
+(integer)
+Largest amount of memory, in bytes, historically used by the young generation
+heap.
+
+`jvm.mem.pools.young.peak_max_in_bytes`::
+(integer)
+Largest amount of memory, in bytes, historically used by the young generation
+heap.
+
+`jvm.mem.pools.survivor.used_in_bytes`::
+(integer)
+Memory, in bytes, used by the survivor space.
+
+`jvm.mem.pools.survivor.max_in_bytes`::
+(integer)
+Maximum amount of memory, in bytes, available for use by the survivor space.
+
+`jvm.mem.pools.survivor.peak_used_in_bytes`::
+(integer)
+Largest amount of memory, in bytes, historically used by the survivor space.
+
+`jvm.mem.pools.survivor.peak_max_in_bytes`::
+(integer)
+Largest amount of memory, in bytes, historically used by the survivor space.
+
+`jvm.mem.pools.old.used_in_bytes`::
+(integer)
+Memory, in bytes, used by the old generation heap.
+
+`jvm.mem.pools.old.max_in_bytes`::
+(integer)
+Maximum amount of memory, in bytes, available for use by the old generation
+heap.
+
+`jvm.mem.pools.old.peak_used_in_bytes`::
+(integer)
+Largest amount of memory, in bytes, historically used by the old generation
+heap.
+
+`jvm.mem.pools.old.peak_max_in_bytes`::
+(integer)
+Highest memory limit, in bytes, historically available for use by the old
+generation heap.
+
+`jvm.threads.count`::
+(integer)
+Number of active threads in use by JVM.
+
+`jvm.threads.peak_count`::
+(integer)
+Highest number of threads used by JVM.
+
+`jvm.gc.collectors.young.collection_count`::
+(integer)
+Number of JVM garbage collectors that collect young generation objects.
+
+`jvm.gc.collectors.young.collection_time_in_millis`::
+(integer)
+Total time in milliseconds spent by JVM collecting young generation objects.
+
+`jvm.gc.collectors.old.collection_count`::
+(integer)
+Number of JVM garbage collectors that collect old generation objects.
+
+`jvm.gc.collectors.old.collection_time_in_millis`::
+(integer)
+Total time in milliseconds spent by JVM collecting old generation objects.
+
+`jvm.buffer_pools.mapped.count`::
+(integer)
+Number of mapped buffer pools.
+
+`jvm.buffer_pools.mapped.used_in_bytes`::
+(integer)
+Size, in bytes, of mapped buffer pools.
+
+`jvm.buffer_pools.mapped.total_capacity_in_bytes`::
+(integer)
+Total capacity, in bytes, of mapped buffer pools.
+
+`jvm.buffer_pools.direct.count`::
+(integer)
+Number of direct buffer pools.
+
+`jvm.buffer_pools.direct.used_in_bytes`::
+(integer)
+Size, in bytes, of direct buffer pools.
+
+`jvm.buffer_pools.direct.total_capacity_in_bytes`::
+(integer)
+Total capacity, in bytes, of direct buffer pools.
+
+`jvm.buffer_pools.classes.current_loaded_count`::
```

Comment:
I think this should be `jvm.classes...` instead of `jvm.buffer_pools.classes...`
```suggestion
`jvm.classes.current_loaded_count`::
```

Reply:
Thanks for catching that.